packages/babel-generator/src/printer.js

```diff
@@ -538,8 +513,17 @@ export default class Printer {
       this._printedCommentStarts[comment.start] = true;
     }
 
-    // whitespace before
-    this.newline(this._whitespace ? this._whitespace.getNewlinesBefore(comment) : 0);
+    // We leave inline flow block comments inline
+    const isNonFlowBlockComment =
+      comment.type === "CommentBlock" &&
+      (
+        (!comment.value.startsWith(":") &&
+        !comment.value.startsWith("flow-include")) ||
```

Comment:
Should I leave the case that only block comments with newlines inside them get newlines around them? Or add newlines around to all block comments?